lib/generators/authentication/templates/rspec/requests/api/sessions_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "Sessions", type: :request do
+  let(:user) { users(:lazaro_nixon) }
+  let(:token) { sign_in_as(user) }
+  let(:default_headers) { { "Authorization" => "Bearer #{token}" } }
+
+  describe "GET #index" do
+    it "returns HTTP success" do
+      get sessions_url, headers: default_headers
+
+      expect(response).to have_http_status(:success)
```

Comment:
And the expects can be like this:
```suggestion
      is_expected.to have_http_status :success
```